### PR TITLE
Fixing time printed in UTC vs local time

### LINKED ISSUE #3225: Many printing routines print time in UTC instead of local time
Try e.g. `verdi comment show` or `verdi show` (but I don't have a full list).

The solution is to use, e.g.,
```python
entity.ctime.astimezone(tzlocal.get_localzone()).strftime('%Y-%m-%d %H:%M')
```
instead of
```python
entity.ctime.strftime('%Y-%m-%d %H:%M')
```

(I don't know if there is a more automated way of doing it).

### PR DESCRIPTION
**Pull request by d-tomerini**
Fixes #3225

I addressed the three instances where it was used by the command line interface.

I did not touch two other case where it might be relevant, the
autogroup names, that are printed from timezone.now()

``aiida.orm.autogroup`` and ``cmdline/commands/cmd_run``, 

where it is assigned as an autogroup name

### CODE REVIEW

File: aiida/cmdline/commands/cmd_data/cmd_remote.py [hunk 1]
```diff
@@ -51,7 +52,7 @@ def remote_ls(ls_long, path, datum):
             mtime = datetime.datetime.fromtimestamp(metadata['attributes'].st_mtime)
```
[the remark this answers is not in the document]
Comment (d-tomerini) [reply aiida/cmdline/commands/cmd_data/cmd_remote.py]: Absolutely - although I feel a little bit bad for triggering the tests again for an 'r'...

[the remark this answers is not in the document]
Comment (CasperWA) [reply aiida/cmdline/commands/cmd_data/cmd_remote.py]: No worries - you have to change some stuff to make the current tests pass anyway :wink: 
From travis (tests, py2.7, both backends):

```shell
ERROR: test_remotels (aiida.backends.tests.cmdline.commands.test_data.TestVerdiDataRemote)
3274
----------------------------------------------------------------------
3275
Traceback (most recent call last):
3276
  File "/home/travis/build/aiidateam/aiida-core/aiida/backends/tests/cmdline/commands/test_data.py", line 457, in test_remotels
3277
    res = self.cli_runner.invoke(cmd_remote.remote_ls, options, catch_exceptions=False)
3278
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/testing.py", line 326, in invoke
3279
    cli.main(args=args or (), prog_name=prog_name, **extra)
3280
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 717, in main
3281
    rv = self.invoke(ctx)
3282
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 956, in invoke
3283
    return ctx.invoke(self.callback, **ctx.params)
3284
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 555, in invoke
3285
    return callback(*args, **kwargs)
3286
  File "/home/travis/build/aiidateam/aiida-core/aiida/cmdline/commands/cmd_data/cmd_remote.py", line 55, in remote_ls
3287
    timezone.localtime(mtime).strftime('%d %b %Y %H:%M')
3288
  File "/home/travis/build/aiidateam/aiida-core/aiida/common/timezone.py", line 86, in localtime
3289
    value = value.astimezone(timezone)
3290
ValueError: astimezone() cannot be applied to a naive datetime
```

[the remark this answers is not in the document]
Comment (CasperWA) [reply aiida/cmdline/commands/cmd_data/cmd_remote.py]: > I guess I can push again when I have a test, like you propose

Yes. And also, personally I'd rather add the 'r' and have the tests run again, than forget about it again and leave it in (if that was the case).

[the remark this answers is not in the document]
Comment (CasperWA) [reply aiida/cmdline/commands/cmd_data/cmd_remote.py]: Finally, concerning the tests, just first check that there aren't any tests that already check the things we want to check. Only if not, perhaps add a test or two :+1:

**Comment by d-tomerini** [comment]
@CasperWA  I corrected the problem I introduced, and the 'r'. Not sure why Travis fails now, anyway

P.S. I still have to add a test

**Comment by CasperWA** [comment]
> @CasperWA I corrected the problem I introduced, and the 'r'. Not sure why Travis fails now, anyway

Cool, thanks :) And concerning Travis - don't know what you mean? All is good now :sweat_smile: (I restarted the failing tests, it was due to networking issues on Travis' side, they are still working on reversing their latest upgrade, see [here](https://www.traviscistatus.com/)).

**Comment by CasperWA** [comment]
After a discussion in the AiiDA Team, we have found that there should probably be a general time parser function of sorts that can be used when printing times, to always show them in the local timezone.

However, for this PR this it is fine for now, and fixes the original issue. Testing will be difficult for this particular thing, so no additional tests will be added.

**Comment by sphuber** [comment]
Thanks @d-tomerini

**Pull request merged**